build.gradle

```diff
@@ -3,26 +3,328 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.2.3'
-        classpath 'de.undercouch:gradle-download-task:1.0'
-        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.4'
+        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:3.1.1'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
-        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
-        classpath 'com.novoda:gradle-android-command-plugin:1.3.0'
-        classpath 'com.github.skhatri:gradle-s3-plugin:1.0.2'
+        classpath 'ch.netzwerg:gradle-release-plugin:1.1.0'
     }
 }
 
-allprojects {
-  apply plugin: 'android-sdk-manager'
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.artifactory'
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'ch.netzwerg.release'
 
-  group = 'io.realm'
-  version = file("${rootDir}/version.txt").text.trim();
-  repositories {
-    jcenter()
-  }
+def currentVersion = file("${projectDir}/version.txt").text.trim();
+
+def getSdk() {
+    if (!System.env.ANDROID_HOME) {
+        throw new GradleException('The ANDROID_HOME environment variable is not set.')
+    }
+    def sdkDir = file(System.env.ANDROID_HOME)
+    if (!sdkDir.directory) {
+        throw new GradleException('The path provided in the ANDROID_HOME environment variable is not a folder.')
+    }
+    return sdkDir
+}
```

Comment:
Does this mean `local.properties` doesn't work anymore? `README.md` needs to be modified as well.